tests/kafkatest/services/kafka_log4j_appender.py

```diff
@@ -32,14 +34,15 @@ def __init__(self, context, num_nodes, kafka, topic, max_messages=-1):
         self.max_messages = max_messages
 
     def _worker(self, idx, node):
-        cmd = self.start_cmd
+        cmd = self.start_cmd(node)
         self.logger.debug("VerifiableKafkaLog4jAppender %d command: %s" % (idx, cmd))
         node.account.ssh(cmd)
 
-    @property
-    def start_cmd(self):
-        cmd = "/opt/kafka/bin/kafka-run-class.sh org.apache.kafka.clients.tools.VerifiableLog4jAppender" \
-              " --topic %s --broker-list %s" % (self.topic, self.kafka.bootstrap_servers())
+    def start_cmd(self, node):
+        cmd = "/opt/%s/bin/" % kafka_dir(node)
```

Comment:
Sure, wasn't suggesting we get a single authoritative source in this patch. Just that this common usage pattern *could* be cleaned up in this patch to use `kafka_file` instead. Either way is fine for this patch though.